docs/networking/diagnostics/inspecting-network-connections-with-netstat/index.md

```diff
@@ -368,221 +729,50 @@ eth0            1      ff02::fb
-tcp        0      0 109.74.193.253:22       79.131.135.223:64917    ESTABLISHED
-tcp        1      0 109.74.193.253:25       193.32.160.136:37752    CLOSE_WAIT
-tcp        1      0 109.74.193.253:25       193.32.160.136:49900    CLOSE_WAIT
-{{< /output >}}
-
-If you want to display both listening and non-listening TCP connections you should add the `-a` command line option in the previous command.
-
-## Displaying UDP Connections only
-
-If you are interested in non-listening UDP connections, you should execute `netstat(8)` with the `-u` option: `netstat -u`.
-
-If you want to display both listening and non-listening UDP connections you should add the `-a` command line option.
-
-    netstat -u -a
-    {{< output >}}
-Active Internet connections (servers and established)
-Proto Recv-Q Send-Q Local Address           Foreign Address         State
-udp        0      0 0.0.0.0:mdns            0.0.0.0:*
-udp        0      0 li140-253.member:syslog 0.0.0.0:*
-udp        0      0 0.0.0.0:60397           0.0.0.0:*
-udp        0      0 0.0.0.0:bootpc          0.0.0.0:*
-udp        0      0 li140-253.members.l:ntp 0.0.0.0:*
-udp        0      0 localhost:ntp           0.0.0.0:*
-udp        0      0 0.0.0.0:ntp             0.0.0.0:*
-udp6       0      0 [::]:mdns               [::]:*
-udp6       0      0 [::]:58949              [::]:*
-udp6       0      0 fe80::f03c:91ff:fe6:ntp [::]:*
-udp6       0      0 2a01:7e00::f03c:91f:ntp [::]:*
-udp6       0      0 localhost:ntp           [::]:*
-udp6       0      0 [::]:ntp                [::]:*
-{{< /output >}}
-
-If you are only interested in non-listening IPv6 UDP connections, you should execute the `netstat -u -6` command.
-
-## Displaying Listening TCP connections only
-
-If you want to display listening TCP connections only, you should execute the next command:
-
-    netstat -l -t
-    {{< output >}}
-Active Internet connections (only servers)
-Proto Recv-Q Send-Q Local Address           Foreign Address         State
-tcp        0      0 localhost:mysql         0.0.0.0:*               LISTEN
-tcp        0      0 0.0.0.0:ssh             0.0.0.0:*               LISTEN
-tcp      101      0 0.0.0.0:smtp            0.0.0.0:*               LISTEN
-tcp6       0      0 [::]:http               [::]:*                  LISTEN
-tcp6       0      0 [::]:ssh                [::]:*                  LISTEN
-tcp6       0      0 [::]:https              [::]:*                  LISTEN
-{{< /output >}}
-
-## Displaying More Output Than Usual
+## Using AWK to process netstat output
 
-### Displaying extended output
+The AWK programming language can help you process `netstat` output and generate handy reports.
 
-The `-e` command line parameter tells `netstat(8)` to display extended output. Therefore, in order to get extended output on the listening TCP ports, you should execute the following command:
+### Showing the Number of Listening Processes Per Username
```

Comment:
Moved `Showing the Number of Listening Processes Per Username` to the first AWK example (from the last), as it was the only one with an explanation of the command, which I thought would be good for a reader to see from the start